Assets/Scripts/GameManager.cs

```diff
@@ -77,6 +77,20 @@ void InitializeOrganisms()
     void Update()
     {
         UpdateSimulation();
+
+        if (Input.GetKeyDown(KeyCode.Space) & !isWatering)
+        {
+          StartCoroutine(Watering());
```

Comment:
VERY FANCY I have not used coroutines in unity, but I have read about them. This seem like a great use of this feature 🔥

Reply:
THANKS I DON'T KNOW WHAT I'M DOING BUT I'M HAVING FUN 👯‍♂️